Pipelines/Templates/run-canvas-tests.yml

```diff
@@ -36,14 +36,14 @@ steps:
       . "$env:POWERSHELLPATH\build-test-automation-urls.ps1"    
       
       $solutionName = "${{parameters.solutionName}}"
-      $canvasAppsPath = "$(Agent.TempDirectory)\$env:SOLUTIONNAME\SolutionPackage\CanvasApps"
+      $canvasAppsPath = "$(Agent.TempDirectory)\${{parameters.solutionName}}\SolutionPackage\CanvasApps"
       $serviceConnection = "${{parameters.serviceConnection}}"
 
-      $microsoftPowerAppsAdministrationPowerShellModule = $env:CoETools_Microsoft_PowerApps_Administration_PowerShell
-      Import-Module $microsoftPowerAppsAdministrationPowerShellModule -Force -RequiredVersion $env:PowerAppsAdminModuleVersion -ArgumentList @{ NonInteractive = $true }
+      $microsoftPowerAppsAdministrationPowerShellModule = '$(CoETools_Microsoft_PowerApps_Administration_PowerShell)'
+      Import-Module $microsoftPowerAppsAdministrationPowerShellModule -Force -RequiredVersion $(PowerAppsAdminModuleVersion) -ArgumentList @{ NonInteractive = $true }
       Add-PowerAppsAccount -TenantID $(connectionVariables.BuildTools.TenantId) -ApplicationId $(connectionVariables.BuildTools.ApplicationId) -ClientSecret $(connectionVariables.BuildTools.ClientSecret)
 
-      Set-CanvasTestAutomationURLs $env:SPNTOKEN $serviceConnection $solutionName $canvasAppsPath $env:EnvironmentId
+      Set-CanvasTestAutomationURLs $(SpnToken) $serviceConnection $solutionName $canvasAppsPath $(BuildTools.EnvironmentId)
```

Comment:
Switching to $() syntax will cause the value to be emitted in the logs.  I've update the PR to ensure SpnToken is a secret